Fix fatal phpcs error: risky-shell-pipe: Shells that use pipes should set the pipefail option

**Describe the bug**

Complain about a shell command with a pipe in it

**To Reproduce**
Steps to reproduce the behavior:

1. Check why PR check is failing

**Expected behavior**

Get check to pass

**Screenshots/Output**

Example from https://github.com/openaustralia/infrastructure/actions/runs/25797532302/job/75778535661?pr=506

```
ANSIBLE_ROLES_PATH=roles:roles/internal:roles/external .venv/bin/ansible-lint roles/internal/ roles/*.yml site.yml 
WARNING: PATH altered to include /home/runner/work/infrastructure/infrastructure/.venv/bin
WARNING  Loading custom .yamllint config file, this extends our internal yamllint config.
WARNING  Overriding detected file kind 'yaml' with 'playbook' for given positional argument: site.yml
WARNING  Listing 1 violation(s) that are fatal
risky-shell-pipe: Shells that use pipes should set the pipefail option
roles/internal/openaustralia/tasks/main.yml:351 Task/Handler: Find latest certbot live dir for each domain

Error: risky-shell-pipe Shells that use pipes should set the pipefail option
You can skip specific rules or tags by adding them to your configuration file:
# .ansible-lint
warn_list:  # or 'skip_list' to silence them completely
  - risky-shell-pipe  # Shells that use pipes should set the pipefail option

Finished with 1 failure(s), 0 warning(s) on 219 files.
make: *** [Makefile:275: ansible-lint] Error 2
Error: Process completed with exit code 2.
```

**Additional context**
Add any other context about the problem here.

--- a/roles/internal/openaustralia/tasks/main.yml
+++ b/roles/internal/openaustralia/tasks/main.yml
@@ -341,11 +341,13 @@
   when: item.key != '000-default'
   with_dict: "{{ apache_sites }}"
 
-
 - name: Find latest certbot live dir for each domain
   tags: sites_available
   shell: |
-    (ls -d /etc/letsencrypt/live/{{ item.value.domain }}-[0-9]* 2>/dev/null ; echo /etc/letsencrypt/live/{{ item.value.domain }}) | sort | tail -1
+    set -o pipefail
+    (ls -d /etc/letsencrypt/live/{{ item.value.domain }}-[0-9]* 2>/dev/null || echo /etc/letsencrypt/live/{{ item.value.domain }}) | tail -1
+  args:
+    executable: /bin/bash
   register: certbot_live_dirs
   changed_when: false
   check_mode: no

